app/adapters/application.js

```diff
@@ -0,0 +1,29 @@
+import DS from 'ember-data';
+// require('electron');
+import database from './database';
```

Comment:
For starters, this would have to import from `../../database`, as that's where the file is actually located within the directory structure. I see that this doesn't fix the entire problem, but I'm still debugging what we actually need to do here.